scripts/languageserver/packages/AbstractTrees

```diff
@@ -1 +0,0 @@
-Subproject commit 74695d4634c2286e20c75accb2739d07344fe56a
```

Comment:
OK I've fixed it back to proper-submodules. I'll set the specific commits after the LS PR is merged/ the Julia release